.github/workflows/test.yml

```diff
@@ -35,3 +35,9 @@ jobs:
         run: poetry install
       - name: Run tests
         run: poetry run pytest
+  dockerize:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Dockerize
+        run: make dockerize
```

Comment:
Let's also make sure this doesn't break in the future and run the `dockerize` target in CI.

Reply:
Nice, I'm not sure I was aware of the fact that you could use `docker` in GitHub Actions! I was somehow presuming they were running in Docker or something... which is odd since they support both macOS and Windows platforms. Seems to be "real" VMs in fact, which makes them amazing fast in terms of startup time, TBH. :astonished: https://github.com/actions/runner-images/blob/main/docs/create-image-and-azure-resources.md

Reply:
:+1: 

It's also used in the `deploy.yml` workflow, that's where I borrowed the concept from :slightly_smiling_face: